node_modules/col-users/tests/util.js

```diff
@@ -71,35 +74,84 @@ var assertMe = module.exports.assertMe = function(me, expectedMe) {
+    assert.strictEqual(me.id, opts.expectedMe.id);
+    assert.strictEqual(me.canvas_user_id, opts.expectedMe.canvas_user_id);
+    assert.strictEqual(me.course_id, opts.expectedMe.course_id);
+    assert.strictEqual(me.canvas_course_role, opts.expectedMe.canvas_course_role);
+    assert.strictEqual(me.canvas_full_name, opts.expectedMe.canvas_full_name);
+    assert.strictEqual(me.bookmarklet_token, opts.expectedMe.bookmarklet_token);
+    assert.strictEqual(me.created_at, opts.expectedMe.created_at);
+    assert.strictEqual(me.updated_at, opts.expectedMe.updated_at);
+    assert.strictEqual(me.points, opts.expectedMe.points);
+    assert.strictEqual(me.share_points, opts.expectedMe.share_points);
+    assert.strictEqual(me.is_admin, opts.expectedMe.is_admin);
 
     // Ensure that all the expected properties on the course property are the same
-    assert.strictEqual(me.course.id, expectedMe.course.id);
-    assert.strictEqual(me.course.canvas_course_id, expectedMe.course.canvas_course_id);
-    assert.strictEqual(me.course.canvas_api_domain, expectedMe.course.canvas_api_domain);
-    assert.strictEqual(me.course.created_at, expectedMe.course.created_at);
-    assert.strictEqual(me.course.updated_at, expectedMe.course.updated_at);
-    assert.strictEqual(me.course.enable_upload, expectedMe.course.enable_upload);
-    assert.strictEqual(me.course.active, expectedMe.course.active);
+    assert.strictEqual(me.course.id, opts.expectedMe.course.id);
+    assert.strictEqual(me.course.canvas_course_id, opts.expectedMe.course.canvas_course_id);
+    assert.strictEqual(me.course.canvas_api_domain, opts.expectedMe.course.canvas_api_domain);
+    assert.strictEqual(me.course.created_at, opts.expectedMe.course.created_at);
+    assert.strictEqual(me.course.updated_at, opts.expectedMe.course.updated_at);
+    assert.strictEqual(me.course.enable_upload, opts.expectedMe.course.enable_upload);
+    assert.strictEqual(me.course.active, opts.expectedMe.course.active);
 
     // Ensure that all the expected properties on the canvas property are the same
-    assert.strictEqual(me.course.canvas.use_https, expectedMe.course.canvas.use_https);
+    assert.strictEqual(me.course.canvas.use_https, opts.expectedMe.course.canvas.use_https);
+
+    // Ensure that all optional properties are the same as the ones for the
+    // expected me object
+    if (me.canvas_image || opts.expectedMe.canvas_image) {
+      assert.strictEqual(me.canvas_image, opts.expectedMe.canvas_image);
+    }
+  }
+};
+
+/**
+ * Assert that a user has all expected properties
+ *
+ * @param  {User}               user                    The user to assert the properties for
+ * @param  {Object}             [opts]                  Optional parameters to verify the user with
+ * @param  {User}               [opts.expectedUser]     The user to which the provided user should be compared
+ * @param  {Boolean}            [opts.expectEveryone]   Whether all users in the course are expected to be returned
```

Comment:
It makes sense for `assertGetUsers` as you're dealing with multiple users. In `assertUser` you're asserting the properties of a user object. I don't think it should know about the context where the user was retrieved from